src/app/modules/evaluate/component/evaluate-search-table/evaluate-search-table.component.ts

```diff
@@ -4,6 +4,8 @@ import { Currency } from '@shared/module/poe/type'
 
 interface Row {
   amount: number
+  priceNumerator: number
+  priceDenominator: number
```

Comment:
do these need to be optional?

Reply:
They're always being filled/set, so I don't see the need to make them optional.